main.css

```diff
@@ -0,0 +1,41 @@
+body {
+  background-color: #efefef;
+}
+
+.container {
+  position: absolute;
+  width: 50%;
+  top: 30%;
+  left: 25%;
+  text-align: center;
+  font-family: Karla;
```

Comment:
declare the font targeting `html` instead. You can declare the base size, weight, and line height for the font as well as the fallback: https://developer.mozilla.org/en-US/docs/Web/CSS/font?v=example

In this case, something like this: `font: 400 16px/1.5 "Karla", sans-serif`